docs/docs/create-redwood-app.md

```diff
@@ -0,0 +1,94 @@
+---
+slug: create-redwood-app
+description:
+---
+
+# Create Redwood App
+
+## Set up for success
+Redwood requires that you're running Node version 18.0.0 or higher.
+
+If you're running Node version 19.0.0 or above, you can still use Create Redwood app, but it may make your project incompatible with some deploy targets, such as AWS Lambdas.
+
+To see what version of Node you're running, you can run the following command in your terminal:
+
+```terminal
+node -v
+```
+
+If you need to update your version of Node or run multiple versions of Node, we recommend installing nvm and have [documentation about how to get up and running.](/docs/how-to/using-nvm)
+
+You also need to have yarn version 1.15 or higher installed. To see what version of yarn you're running, you can run the following command in your terminal:
+
+```terminal
+yarn -v
+```
+
+To upgrade your version of yarn, [you can refer to the yarn documentation](https://yarnpkg.com/getting-started/install).
+
+## What you can expect
+
+### Select your preferred language
+Options: TypeScript or JavaScript
+
+If you choose JavaScript, you can always [add TypeScript later](/docs/typescript/introduction#converting-a-javascript-project-to-typescript).
+
+### Do you want to initialize a git repo?
+Options: yes (default) or no
+
+If you mark "yes", then it will ask you to **Enter a commit message**. The default is message is "Initial commit."
+
+You can always initialize a git repo later and add a commit message by running the following commands in your terminal:
+
+```terminal
+git init
+git add .
+git commit -m "Initial commit"
+```
+
+If you're new to git, Amy Dutton has a great playlist on YouTube: [git for Beginners](https://www.youtube.com/playlist?list=PLrz61zkUHJJFmfTgOVL1mBw_NZcgGe882)
```

Comment:
I added this link to help anyone that wants to get started with git. However, it won't hurt my feelings if we want to take this out.

Reply:
How about `If you're new to git, here's a recommended playlist on YouTube: ...`

It's a great playlist! It's the downstream doors (re: potential self-promotion) this could open that I'd prefer to remain closed.